private/buf/bufcurl/reflection_resolver.go

```diff
@@ -274,9 +274,7 @@ func (r *reflectionResolver) fileByNameLocked(name string) ([]*descriptorpb.File
 func descriptorsInResponse(resp *reflectionv1.ServerReflectionResponse) ([]*descriptorpb.FileDescriptorProto, error) {
 	switch response := resp.MessageResponse.(type) {
 	case *reflectionv1.ServerReflectionResponse_ErrorResponse:
-		// TODO: ideally, this would be a wire error
-		//   see https://github.com/bufbuild/connect-go/issues/420
```

Comment:
This TODO can now be addressed thanks to https://github.com/bufbuild/connect-go/pull/437